### PR TITLE
store_youtube_channel_info_MOD.js

### PR DESCRIPTION
**Pull request by TheRealPikachu**
**Please describe the changes this PR makes and why it should be merged:**

current features: store youtube channel ID 
going to add more features soon 

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by TheRealPikachu** [comment]
This was supposed to be a draft PR

File: store_youtube_channel_info_MOD.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      } else {
+        console.log(`Channel ID: ${reezult.id}`);
+      }
+    } else {
+      console.log('Invalid youtube channel URL');
+    }
+
+    switch (INFO) {
+      case 0:
+        result = reezult.id
```
Comment (noahalma) [inline store_youtube_channel_info_MOD.js:75]: Where did you define result

**Comment by TheMonDon** [comment]
I think this would be better suited to be put inside the store youtube info rather than a separate mod.

**Comment by TheRealPikachu** [comment]
How?

**Pull request closed without merging**